### ISSUE TITLE
Versions Differ Error

### ISSUE THREAD

**Issue by oliverdelancey** [joaotavora/sly#480]
Whenever I open a Common Lisp buffer, Emacs gives this warning:
```
[sly] Versions differ: 1.0.43 (sly) vs. 1.0.43^M (slynk). Continue? (y or n)
```
It seems that something is up with file encoding somewhere? I assume that `^M` has something to do with Windows vs *nix line-endings.

I took a cursory look through the source code for sly/slime in my `.emacs.d` but I could not find where version numbers are defined. I assume I just need to reformat the line endings in a file, but I'm not sure which one.

OS: Windows 11
Emacs: 27.2
Sly/Slime: 1.0.43

**Comment by joaotavora** [comment]
version numbers are defined in the `sly.el` file, if I'm not mistaken.

**Comment by oliverdelancey** [comment]
I did this to all `*.el` and `*.lisp` files in the sly and slynk folders (probably a bit overkill to do it for all, but it worked):

1. Converted line endings from Windows CR-LF to Unix LF.
2. Replaced rogue line feed characters with newline characters. (Perhaps unnecessary but I like to be thorough).
3. Ran `doom build` to recompile sly.

This seems to have fixed the issue, although it is puzzling how it occurred in the first place.
